EMIP/1-1000/EMIP00616.xml

```diff
@@ -94,7 +94,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <msItem xml:id="ms_i10"><locus from="38v" to="40v" facs="041"/><title type="complete" ref="LIT5888AsmatPrayer"> Asmat Prayer against evil spirit, evil eye, epilepsy, and Legewon</title><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> በስሙ ፡ ለእግዚአብሔር ፡ አብ ፡ በስሙ ፡ ለእግዚአብሔር ፡ ወልድ ፡ በስሙ ፡ ለእግዚአብሔር ፡ መንፈስ ፡ ቅዱስ ፡ ታኦስ ፡ አዝያስ ፡ ወለሚልካያስ ፡ አክሴፍር ፡ አሐፌፌክር ፡ ወወጁን ፡ ዘሐጁን ፡ ዋክ ፡ ፍልፍልማየን ፡ አብሲትር ፡ አፍርርዋቅ ፡ ወገጹ ፡ ዘፍጹም ፡ በጽልመት፡</incipit>
                   <textLang mainLang="gez">Gǝ‘ǝz</textLang>
                 </msItem>
-                  <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT3017RepCh297"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
+                 <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT6161BakwelluH"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
```

Comment:
As far as I understand, at the end of the quotation from the incipit you do not need <gap>. It is used, for instance, if you the text is interrupted with the quotation.

Reply:
Thanks for clarifying this, I had wondered.